app/assets/javascripts/models/data/service_options.js.coffee

```diff
@@ -119,7 +118,12 @@ ns.ServiceOptions = do (ko, edsc = @edsc, KnockoutModel = @edsc.models.KnockoutM
                 if !accessMethodMatched && matches[1] == jsonObj.collection_id && parseInt(matches[2], 10) == index
                   accessMethodMatched = true
                   @loadForm(true) if jsonObj.type == 'service'
-                  ko.applyBindingsToNode(echoformContainer, {echoform: this})
+                  # After the binding is complete, toggle select elements to trigger validation lest they be pruned
+                  $.when(ko.applyBindingsToNode(echoformContainer, echoform: this)).done ->
```

Comment:
@mightynimble - It doesn't need to work for "South Polar Stereographic".  What that line of code does is it checks to see if the select contains "North Polar Stereographic" as an option, and the projection select always has that as an option.  

To rephrase to pseudocode, it says "Find all selects on the page that contain 'North Polar Stereographic' as one of its options'

And it works.  It's one line of code.  Your solution is interesting, but why would I replace one line of functional code that's been tested and shown to work with many lines of code that have not?